src/main/java/Attendance.java

```diff
@@ -0,0 +1,18 @@
+import java.util.ArrayList;
+
+public class Attendance {
```

Comment:
No class is pointing to attendance at the moment which will give the impression that this class can be instantiated anywhere in the program. This can be confusing since we want to associate attendance with a particular event.